assets/js/workflow-diagram/nodes/Node.tsx

```diff
@@ -45,7 +45,7 @@ const Label: React.FC<LabelProps> = ({ children, hasErrors = false }) => {
   if (children && (children as any).length) {
     return (
       <p
-        className={`line-clamp-2 align-left text-m max-w-[275px] text-ellipsis overflow-hidden ${textColorClass}`}
+        className={`line-clamp-2 align-left text-m max-w-[120px] text-ellipsis overflow-hidden ${textColorClass}`}
```

Comment:
checking this out. looks a bit tricky. 
if I remember clearly. this value has to be inline with the value used for dagre calculations.

Hence if this has reduced. the value for width in dagre calculations can also be reduced a bit to make good use of the space if we want to. 
<img width="646" alt="Screenshot 2025-06-24 at 12 35 09 PM" src="https://github.com/user-attachments/assets/963e97f3-5ead-4d60-b8e1-c5dbb50a8bd4" />


That would close up the space between the node with title "This is a sill...." and the one next to it

cc: @josephjclark

Reply:
This is fine - I needed to reduce the label width. It's breaking on whitespace in your screen and then sliding the rest.

We can increase the label length a bit if we want to.

Meanwhile the dagre layout sizes and distancing are controlled elsewhere